SimpleRss/SimpleRss/Views/FeedViewCell.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  FeedViewCell.swift
+//  SimpleRss
+//
+//  Created by Voldem on 11/19/18.
+//  Copyright © 2018 Vladimir Koptev. All rights reserved.
+//
+
+import UIKit
+
+class FeedViewCell: UITableViewCell {
+    
+    @IBOutlet weak var PreviewImageView: UIImageView!
```

Comment:
Please rename :)
All variables, properties and methods start with lowercased letter.